package.json

```diff
@@ -112,7 +112,7 @@
     "release": "standard-version",
     "test:all": "npm run lint && npm run format && npm run test",
     "test:perf": "cross-env NODE_ENV=production babel-node test/perf",
-    "test:watch": "jest --watch",
+    "test:watch": "cross-env NODE_ICU_DATA=./node_modules/full-icu jest --watch",

```

Comment:
This is what tricked me. Now the polyfilled data is loaded also when running jest in watch mode.

Reply:
then u prob don't need to mock `intl-locales-supported` right?